google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -91,6 +91,10 @@
 public class LoggingImplTest {
 
   private static final String PROJECT = "project";
+  private static final String PROJECT_OVERRIDE = "projectoverride";
```

Comment:
I think that the use of "override" word in the name of the constant introduce a logic that may misguide or mislead a reader. Also it is useful when the name of the constant defines the role of the constant value and not only its content. For example, `PROJECT` constant is used as a project id and will be more readable as `PROJECT_ID` or `DEFAULT_PROJECT_ID`.
I recommend to address all relevant constant names accordingly.

Reply:
Renamed override.